website/content/docs/v0.14/Guides/logging.md

```diff
@@ -35,8 +35,28 @@ $ talosctl -n 172.20.1.2 logs machined
 [...]
 ```
 
+Container logs for Kubernetes pods can be retrieved with `talosctl logs -k` command:
+
+```sh
+$ talosctl -n 172.20.1.2 containers -k
+NODE         NAMESPACE   ID                                                 IMAGE                                                         PID    STATUS
+172.20.1.2   k8s.io      kube-system/kube-flannel-dk6d5                     k8s.gcr.io/pause:3.5                                          1329   SANDBOX_READY
+172.20.1.2   k8s.io      └─ kube-system/kube-flannel-dk6d5:install-cni      ghcr.io/talos-systems/install-cni:v0.7.0-alpha.0-1-g2bb2efc   0      CONTAINER_EXITED
+172.20.1.2   k8s.io      └─ kube-system/kube-flannel-dk6d5:install-config   quay.io/coreos/flannel:v0.13.0                                0      CONTAINER_EXITED
+172.20.1.2   k8s.io      └─ kube-system/kube-flannel-dk6d5:kube-flannel     quay.io/coreos/flannel:v0.13.0                                1610   CONTAINER_RUNNING
+172.20.1.2   k8s.io      kube-system/kube-proxy-gfkqj                       k8s.gcr.io/pause:3.5                                          1311   SANDBOX_READY
+172.20.1.2   k8s.io      └─ kube-system/kube-proxy-gfkqj:kube-proxy         k8s.gcr.io/kube-proxy:v1.23.0-rc.0                            1379   CONTAINER_RUNNING
+
+$ talosctl -n 172.20.1.2 logs -k
```

Comment:
missing container name (`talosctl -n 172.20.1.2 logs -k <container>`)? Or do we get all logs in this case?

Reply:
nope, it's copy-paste error :facepalm:

Reply:
fixed